app/controllers/api/v1/jwt_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Api::V1::JwtController < Doorkeeper::ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  before_action :doorkeeper_authorize!
+
+  respond_to :json
+
+  def create
+    head :bad_request and return unless doorkeeper_token.resource_owner_id.nil?
```

Comment:
Can the `unless nil?` be rewritten as `if present?`
```suggestion
    head :bad_request and return if doorkeeper_token.resource_owner_id.present?
```